AdobeMediaGallery/Model/Asset/Command/GetById.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeMediaGallery\Model\Asset\Command;
+
+use Magento\AdobeMediaGalleryApi\Api\Data\AssetInterface;
+use Magento\AdobeMediaGalleryApi\Api\Data\AssetInterfaceFactory;
+use Magento\AdobeMediaGalleryApi\Model\Asset\Command\GetByIdInterface;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * Class GetById
+ */
+class GetById implements GetByIdInterface
+{
+    private const TABLE_ADOBE_MEDIA_GALLERY = 'adobe_media_gallery';
```

Comment:
References to `Adobe` should be avoided in this module (except of temporary module name) as it's going to be part of Magento core
```suggestion
    private const TABLE_MEDIA_GALLERY_ASSET = 'media_gallery_asset';
```